README.md

```diff
@@ -950,7 +950,7 @@ Table of Contents
 ## DBaaS
 
    * [airtable.com](https://airtable.com/) — Looks like a spreadsheet, but it's a relational database, unlimited bases, 1,200 rows/base and 1,000 API requests/month
-   * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with 5GB free tier
+   * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with [40GB free tier](https://docs.datastax.com/en/astra/docs/datastax-astra-database-limits.html)
```

Comment:
Please set the link to https://www.datastax.com/products/datastax-astra/  which demonstrates the features etc - better landing place for people from this list.

Then I think https://www.datastax.com/products/datastax-astra/pricing is probably better if you want to link to the plans

Reply:
Done!